### ISSUE TITLE
Stay for some time if Lured Pokestop found

### ISSUE THREAD

**Issue by dswtech** [finndev/PokeBuddy#203]
Maybe You can add feature that the bot will stay for 5 to 15 minutes if find lured Pokestop #

**Comment by ShupengHu** [comment]
Agreed, There is no pokemon near many normal pokestops but lots of pokemons near lured pokestops.
